src/metrics/peerstore.go

```diff
@@ -359,6 +360,69 @@ func (c *PeerStore) ExportToCSV(filePath string) error {
 	return nil
 }
 
+// Function that save the raw Data of all the peers to a db (Using BoltDB)
+// Using the peerID as a key, and the Peer struct marshalled into JSON as a value
+func (c *PeerStore) PersistToDb(path string) error {
+	db, err := bolt.Open(path, 0600, nil)
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	err = db.Update(func(tx *bolt.Tx) error {
+		b, err := tx.CreateBucketIfNotExists([]byte("PeerStore"))
+		if err != nil {
+			return err
+		}
+		
+		c.PeerStore.Range(func(k, val interface{}) bool {
+			peerData := val.(Peer)
+			var buf []byte
+			buf, err = json.Marshal(peerData)
+			if err != nil{
+				return false
+			}
+			
+			err = b.Put([]byte(peerData.PeerId), buf)
+			
+			return err == nil
```

Comment:
The go static check suggested me this change. The range function takes as an input a function that return a bool.
According to the range function documentation, if the function returns false it will stop the iteration